backend/src/main/java/reviewme/review/controller/ReviewController.java

```diff
@@ -12,18 +15,22 @@
 import reviewme.review.dto.response.ReviewResponse;
 import reviewme.review.service.ReviewService;
 
+@Tag(name = "리뷰 관리")
 @RestController
 @RequiredArgsConstructor
 public class ReviewController {
 
     private final ReviewService reviewService;
 
+    @Operation(summary = "리뷰 등록", description = "리뷰 작성 정보를 받아 리뷰를 등록한다.")
+    @ApiResponse(responseCode = "201")
```

Comment:
Operation 하나로 가져가면 어떤가요? 가독성 측면은 원래 코드가 나으려나요? 
```suggestion
    @Operation(
            summary = "리뷰 등록",
            description = "리뷰 작성 정보를 받아 리뷰를 등록한다.",
            responses = @ApiResponse(responseCode = "201")
    )
```

Reply:
가독성이 훨씬 좋네요👍